src/lib/browser/BrowserPage.ts

```
@@ -1,16 +1,18 @@
 import { z } from 'zod';
+// pdf.js ESM
+import { getDocument, GlobalWorkerOptions } from 'pdfjs-dist'
 import { type BrowserContextConfig } from './BrowserContext';
```

ESLint

[error] 'BrowserContextConfig' is defined but never used.

(@typescript-eslint/no-unused-vars)